lein-template/src/leiningen/new/duct.clj

```diff
@@ -108,7 +114,14 @@
        (map (fn [[path temp]] [path (render-resource data temp)]))))
 
 (defn generate-project [{:keys [vars templates dirs] :as profile}]
-  (let [data  (merge vars (select-keys profile [:deps :dev-deps]))
+  (let [data (->
+               (merge vars (select-keys profile [:deps :dev-deps
+                                                 :modules :profile-base :profile-dev
+                                                 :repl-options]))
+               (update :modules formatting/parse-map)
+               (update :profile-base formatting/parse-map)
+               (update :profile-dev formatting/parse-map)
+               (update :cascading-routes formatting/parse-cascading-routes))
```

Comment:
Why did you make cascading routes a top level key instead of just a var? My thought is that we don't want to give such a specific key special behaviour.

Reply:
It is just a var. Take note that this comes after the `(merge vars ...)` function.

Reply:
Ah, well adding special behaviour to a var is even worse :) - vars should behave consistently.

Reply:
Oh, I see. So that collection of strings holding "#ig/ref route" should be completely prepared by the template, not by duct, right?